client/blocks/product-purchase-features-list/jetpack-anti-spam.jsx

```diff
@@ -12,13 +12,15 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 
-export default localize( ( { translate } ) => {
+export default localize( ( { selectedSite, translate } ) => {
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
 				icon={ <img alt="" src="/calypso/images/illustrations/jetpack-spam.svg" /> }
 				title={ translate( 'Spam Filtering' ) }
 				description={ translate( 'Spam is automatically blocked from your comments.' ) }
+				buttonText={ translate( 'View settings' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'View Settings' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).